packages/frontend/core/src/components/blocksuite/block-suite-editor/specs/custom/database-block.ts

```diff
@@ -46,21 +53,23 @@ function createCopyLinkToBlockMenuItem(
     name: 'Copy link to block',
     icon: LinkIcon({ width: '20', height: '20' }),
     hide: () => {
-      const { editor } = framework.get(EditorService);
-      const mode = editor.mode$.value;
+      const pageId = model.doc.id;
+      const docsService = framework.get(DocsService);
+      const mode = docsService.list.getPrimaryMode(pageId) || DocMode.Page;
```

Comment:
PrimaryMode !== editor mode

Reply:
should use editor mode here?

Reply:
Try it before, `EditorService` will be missing

Reply:
Let me try it again.